stream_alert/rule_processor/rules_engine.py

```diff
@@ -19,7 +19,7 @@
 
 from stream_alert.rule_processor import LOGGER
 from stream_alert.rule_processor.threat_intel import StreamThreatIntel
-from stream_alert.shared import NORMALIZATION_KEY
+from stream_alert.shared import resources, NORMALIZATION_KEY
```

Comment:
In the alert processor, these are imported in the other order. Not sure which way is the "proper" alphabetization, but let's make it the same either way

Reply:
good catch - this was a lazy find/replace since I used to have `resources` named `common` but switch the naming convention